libs/SmartSync/SmartSync.xcodeproj/project.pbxproj

```diff
@@ -620,6 +626,8 @@
 				CEB808B7199AFD1900EF2F9D /* SFSmartSyncMetadataManager.m */,
 				4F22155A19DF4BAD00FF2D26 /* SFSmartSyncSyncManager.h */,
 				4F22155B19DF4BAD00FF2D26 /* SFSmartSyncSyncManager.m */,
+				FDCEC1FBF745182967AD6BA9 /* SalesforceSDKManagerWithSmartSync.m */,
```

Comment:
This one is not named like Android's. Instead it's consistent with the SmartStore SDK manager (called SalesforceSDKManagerWithSmartStore on iOS).

Reply:
Any reason to not make this consistent with `Android`? Or all `SDKManager*` classes consistent?

Reply:
We can name the SmartSync one anything we want.
What about the SmartStore one? Should we rename it in 6.0 or leave it alone?
If we leave it alone, should we have the SmartSync one be named similarly?

Reply:
If we want to change names to make them consistent, we should probably do that now in 6.0. I feel like it would be nice to make the names as consistent as we can between iOS and Android - devs would relate to components better. Maybe we rename the manager classes to be consistent on `SmartSync` and `SmartStore` too?